src/applications/accredited-representative-portal/components/common/Header/Navigation.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { isMobile } from 'react-device-detect';
```

Comment:
This mobile detection appears to function differently from the fundamental approach to responsive design based on media queries of the viewport.

I couldn't get the mobile design to kick in using Chrome dev tools responsive toolbar:
<img width="528" alt="Screenshot 2024-12-10 at 12 03 57 AM" src="https://github.com/user-attachments/assets/d2ddfbca-31e9-4f4d-a68e-0ab08fa826dc">

When I simply resize my window, Chrome doesn't let me go below a certain width, and it still looks fine at that minimum width:
<img width="497" alt="Screenshot 2024-12-10 at 12 05 23 AM" src="https://github.com/user-attachments/assets/e74b8b94-48d7-4507-b577-a96c378180ef">

Indeed the documentation for this library `react-device-detect` [recommends against](https://github.com/duskload/react-device-detect?tab=readme-ov-file#when-to-use-this-library) this being a solution for responsive design.

Thoughts?

Reply:
if you look at the original file MobileLogoRow (it is one of the deleted files) you'll find that we were passing a prop called isMobile which always turned out to be true. I found this hook in the existing codebase and figured we were ok to use it. Will check out the link this afternoon when I get back.

Reply:
removed use of `react-device-detect`

Reply:
Yeah the previous components were a doozy. I wasn't intimately familiar with them, I think they may have been hiding and showing the mobile vs non-mobile sets of components based on media queries that would toggle `display` properties.